introduction.md

```diff
@@ -2,3 +2,10 @@ Hi Guys,
 My name is Ashish. And I work with LAMP stack technologies.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?